src/components/molecules/AiInsightsTab/AiInsightsTab.tsx

```diff
@@ -1,19 +1,40 @@
+import {ReactComponent as LockedIcon} from '@assets/lockedIcon.svg';
+
+import {Button, Text} from '@custom-antd';
+
 import {Execution} from '@models/execution';
 
-import {usePluginState} from '@plugins/pluginHooks';
+import Colors from '@styles/Colors';
+
+import {externalLinks} from '@utils/externalLinks';
+
+import {AiInsightContainer, AiInsightContent} from './AiInsightsTab.styled';
 
 interface AiInsightsTabProps {
   execution: Execution;
   test: any;
 }
 
 const AiInsightsTab = () => {
-  const [{execution}] = usePluginState<AiInsightsTabProps>('testExecutionTabs');
   return (
-    <div>
-      This is a cloud only feature. Please log in to cloud in order to use this feature for id
-      {execution.id}
-    </div>
+    <AiInsightContainer>
+      <AiInsightContent>
+        <LockedIcon />
+        <Text className="big bold"> This feature is available only in Testkube Cloud.</Text>
+        <Text color={Colors.slate400}>
+          Start using Testkube Cloud to get AI insights for your test executions, as well as other exclusive features.
+          Learn more
```

Comment:
Should it be there? Or maybe there is some text missing?

```suggestion
```

Reply:
should be a link to an explanatory page. Since we don't have the page now, this is a placeholder

Reply:
good point. will change it to an empty link. so that we don't forget about it

Reply:
For now it could just redirect to Cloud 🙂